src/components/atoms/Button/Button.tsx

```diff
@@ -92,23 +92,23 @@ export const buttonStyle = css<IButtonProps>`
   }
 `;
 
-const SButton = styled.button<IButtonProps>`
+const ButtonWrapper = styled.button<ButtonProps>`
   ${buttonStyle}
 `;
 
-const Button: React.FC<IButtonProps> = props => {
+const Button: React.FC<ButtonProps> = props => {
   const { children, loading, styling = 'primary', disabled, ...rest } = props;
   const isLoading = styling !== 'link' && loading;
 
   return (
-    <SButton styling={styling} loading={isLoading} disabled={isLoading || disabled} {...rest}>
+    <ButtonWrapper styling={styling} loading={isLoading} disabled={isLoading || disabled} {...rest}>
```

Comment:
[Question] Why not just `<Wrapper>` since you are in the button file :)

Reply:
I think it's easier to understand the component you are wrapping. But I don't care tbh